pkgs/by-name/ru/rust-stakeholder/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage {
+  pname = "rust-stakeholder";
+  version = "unstable-2025-03-15";
```

```suggestion
  version = "0-unstable-2025-03-15";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning